docs/getting_started/advanced_setup_guides.md

```diff
@@ -157,9 +164,60 @@ Keep in mind, if you execute
 ```bash
 docker-compose down
 ```
-you will loose all your datasets in Rubrix!
+you will lose all your datasets in Rubrix!
 :::
 
+### Persisting ElasticSearch data
+To avoid losing all the data when the docker-compose/server goes down, you can add some persistence by mounting a local
+volume in docker compose and granting the required permissions. 
```

Comment:
Now, we could remove references to local volume and required permissions, right?
```suggestion
To avoid losing all the data when the docker-compose/server goes down, you can add some persistence by mounting a 
volume in the docker compose. 
```

Reply:
Yeah, agree :D